src/core/execute.c

```diff
@@ -739,27 +739,36 @@ static int ask_for_confirmation(const char *vc, const char *cmdline) {
                 goto restore_stdio;
         }
 
-        r = ask_char(&c, "yfs", "Execute %s? [Yes, Fail, Skip] ", e);
-        if (r < 0) {
-                write_confirm_error_fd(r, STDOUT_FILENO);
-                r = 1;
-                goto restore_stdio;
-        }
+        for (;;) {
+                r = ask_char(&c, "yfs?", "Execute %s? [y/f/s/? for help] ", e);
```

Comment:
i find this misleading too, I must say... I'd read the "/" as an character of equivalency, hence would automatically think that "y", "f", "s" and "?" are all equivalent, which they of course are not.

I think a better choice would be to use ANSI colors here, and underline, highlight or colorize the relevant letters... I.e. [**Y**es, **F**ail, **S**kip, **H**elp]